admin-dev/themes/new-theme/js/pages/customer/index.ts

```diff
@@ -86,4 +86,45 @@ $(() => {
   new LinkableItem();
 
   new FormSubmitButton();
+
+  // Scroll to the block
+  scrollToBlock();
```

Comment:
@Progi1984 this feature seems interesting for other grids as well no? Maybe it should be integrated inside the `table-sorting` component instead so that it benefits everyone and avoids duplicate code.

If you don't want to impact other pages too much, which is understandable, then it can be integrated in the component via an option in the `SortingExtension` which would then be passed to the `TableSorting` component. 

This way you can keep this new feature disabled by default on every other page and only enable it for this page for now (so we can test and enable it by default on all other pages in another PR).

Reply:
@Progi1984 This is optional which is why I didn't block nor approved the PR (it was already approved) but just commented it.

Reply:
(If I have feedback from QA on this PR, I will make it :+1:).

Reply:
Damn, QA is green 😅